lib/table.js

```diff
@@ -124,13 +124,13 @@ Table.prototype.insert = function(docs, options, query) {
   var newDoc;
   if (util.isSequence(docs)) {
     for(var i=0; i<docs.length; i++) {
-      newResult = this._singleInsert(docs.get(i), options, query);
-      util.mergeWriteResult(result, newResult);
+      newDoc = this._singleInsert(docs.get(i), options, query);
```

Comment:
there's an undefined/ununsed variable here `generatedKeys` that seems important?